rpcs3/Emu/Cell/Modules/StaticHLE.cpp

```diff
@@ -32,7 +32,7 @@ statichle_handler::~statichle_handler()
 
 bool statichle_handler::load_patterns()
 {
-	for (int i = 0; i < shle_patterns_list.size(); i++)
```

Comment:
Actually abuse of auto, and it will become int anyway. Use u32 probably.

Reply:
That's how warnings show up. And anything labeled size should be size_t, no?

Reply:
It should be `decltype(thing)::size_type`, but no one sane would write that, so it is what it is.